src/chest.py

```diff
@@ -0,0 +1,88 @@
+import time
+import os
+
+from logger import Logger
+from template_finder import TemplateFinder
+from screen import Screen
+from char import IChar
+from config import Config
+
+class Chest:
+    def __init__(self, char: IChar, template_finder: TemplateFinder, template: str = None):
+        config = Config()
+        self._screen = Screen(config.general["monitor"])
+        self._char = char
+        self._template_finder = template_finder
+        self._folder_name = "chests"
+        self._min_score = 0.8
+        # load all templates
+        self._templates = []
+        for filename in os.listdir(f'assets/{self._folder_name}/{template}'):
+            filename = filename.lower()
+            if filename.endswith('.png'):
+                chest = filename[:-4].upper()
+                self._templates.append(chest)
+                
+    def open_up_chest(
+        self,  
+        time_out: float = 8,
+        threshold: float = 0.8
+    ) -> float:
+        score = 100
```

Comment:
template match scores are in float range of [0, 1]. That 100 confuses me.

Reply:
will replace it for 1